.github/workflows/python.yml

```diff
@@ -62,6 +64,8 @@ jobs:
         uses: PyO3/maturin-action@v1.44.0
         with:
           manylinux: auto
+          # Don't use a docker container for building the wheels
+          container: off
```

Comment:
OH, that would make sense. The CI seems to work on this PR, but I'm guessing it would fail to Release? 🤔  Is there a way to test that?

Reply:
CI works for these wheels because we're not testing them in their restricted environments. This change would break these wheels at runtime when they attempt to link to system libs.

Reply:
Also! I hadn't seen #276 - looks like there's some overlap there with this PR! My apologies 😓. Anything I can do there to make that easier?

Reply:
No problem! I can rebase those changes on the changes in this PR. I still have to fix the `sdist` build.

If you fix the manylinux build as suggested above, we can merge it. This PR also unblocks #281.

Reply:
Just fixed the manywheels build - your code just worked the first time, so seems to be right. Does it look good to you?